Content.Server/Chat/Systems/ChatSystem.cs

```diff
@@ -221,6 +222,15 @@ public void TrySendInGameICMessage(
             message = message[1..];
         }
 
+        // Delta-V
+        // This needs to happen after prefix removal to avoid bug
+        if (desiredType == InGameICChatType.Speak&&HasComp<HushedComponent>(source))
```

Comment:
put spaces before and after `&&` because it looks awful

Reply:
my bad!